app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/ProjectPledgeButtonAndFragmentContainer.kt

```diff
@@ -291,7 +292,9 @@ fun ProjectPledgeButtonAndFragmentContainer(
                                         newPaymentMethodClicked = onAddPaymentMethodClicked,
                                         isLoading = isLoading,
                                         onDisclaimerItemClicked = onDisclaimerItemClicked,
-                                        onAccountabilityLinkClicked = onAccountabilityLinkClicked
+                                        onAccountabilityLinkClicked = onAccountabilityLinkClicked,
+                                        isPlotEnabled = environment?.featureFlagClient()
+                                            ?.getBoolean(FlagKey.ANDROID_PLEDGE_OVER_TIME) ?: false,
```

Comment:
should check for pledge reason here as well

Reply:
Actually for late pledges flow should always be false.  Is gonna be an available payment option only for new pledges while in campaign

Reply:
changin it rn